src/IssueViz.Security/Commands/ShowToolWindowCommand.cs

```diff
@@ -20,48 +20,48 @@
 
 using System;
 using System.ComponentModel.Design;
+using System.Threading.Tasks;
 using Microsoft.VisualStudio.ComponentModelHost;
 using Microsoft.VisualStudio.Shell;
 using SonarLint.VisualStudio.Core;
 using SonarLint.VisualStudio.Integration;
-using SonarLint.VisualStudio.IssueVisualization.Security.UI.HotspotsList;
-using Task = System.Threading.Tasks.Task;
 
 namespace SonarLint.VisualStudio.IssueVisualization.Security.Commands
 {
-    internal sealed class HotspotsToolWindowCommand
+    internal sealed class ShowToolWindowCommand
```

Comment:
The GitHub is showing the new class as a modification to one of the existing ones. Still, perhaps that makes it easier to see the differences between them.